### PR TITLE
x11-plugins/pidgin-telegram: add LibreSSL support

### PR DESCRIPTION
**Pull request by steils**
Closes: https://bugs.gentoo.org/666562
Package-Manager: Portage-2.3.51, Repoman-2.3.12
Signed-off-by: Stefan Strogin <stefan.strogin@gmail.com>

### CODE REVIEW

**Comment by steils** [comment]
Oh, forgot to return the reverted USE flag...

File: x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild [hunk 1]
```diff
@@ -10,13 +10,16 @@ SRC_URI="https://github.com/majn/telegram-purple/releases/download/v${PV}/telegr
 LICENSE="GPL-2+"
 KEYWORDS="~amd64 ~x86"
 SLOT="0"
-IUSE="gcrypt +nls +webp"
+IUSE="gcrypt libressl +nls +webp"
```
Comment (Zlogene) [inline x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild]: second time I tell you to revbump the ebuild once you change the IUSE, if you cannot follow what you've been told by at least 2 different people, please do not contribute

Comment (mgorny) [reply x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild]: So here's third voice that says you don't need to revbump when adding USE flags to ~arch package.

Comment (Zlogene) [reply x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild]: @mgorny how did you count? ;) Amy and I already told him to change the revision for scheme things

Comment (steils) [reply x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild]: Hi @Zlogene, thanks for the comment. I try to follow what's written in devmanual, and existing Gentoo practices. For sure sometimes I'm mistaken, I'm still new to Gentoo. You and Amy Liffey asked to revbump the ebuild for dev-scheme/gambit (https://github.com/gentoo/gentoo/pull/9929), and I did as you asked. And I understand the logic: always revbump stable ebuilds, even with a slightest change (although it's not _obvious_ from the devmanual [for me]).

Yet I sent about 5 patches that add USE=libressl to unstable packages without changing revisions, approved by like 2 or 3 different developers. There are much more in the tree from other contributors, so I think it's a common practice. And from the devmanual it is not obvious [for me] why to revbump every time I add a USE flag that makes no difference to systems with the _unstable_ package already installed.

If you still insist that the revision should be changed in this case, let's suppose you are right. But maybe it's a good idea to do something with this bug then? https://bugs.gentoo.org/223153

PS. Excuse me, but I'm not going to stop contributing. I like it.

**Comment by steils** [comment]
The patch was applied upstream (branch dev-1.4.0): https://github.com/majn/telegram-purple/commit/2bf0d8a49aa527a1731eb7548de245a6e34173ef :-)

File: x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild [hunk 2]
```diff
@@ -26,6 +29,8 @@ DEPEND="
 	virtual/pkgconfig
```
Comment (ConiKost) [inline x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild]: Could you move `virtual/pkgconfig` to BDEPEND, as it's EAPI=7? Thanks!

Comment (steils) [reply x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1-r1.ebuild]: Thank you! Sure. Done.
Also changed the revision as Mikle wants :)

**Comment by mgorny** [comment]
@ConiKost, please re-review.

**Pull request closed without merging**